[2025-07-14T15:26:53.601+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-14T15:26:53.625+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:26:47.949086+00:00 [queued]>
[2025-07-14T15:26:53.640+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:26:47.949086+00:00 [queued]>
[2025-07-14T15:26:53.667+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-07-14T15:26:53.690+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task_2> on 2025-07-14 15:26:47.949086+00:00
[2025-07-14T15:26:53.702+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mon_premier_dag', 'task_2', 'manual__2025-07-14T15:26:47.949086+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/mon_premier_dag.py', '--cfg-path', '/tmp/tmptdlo011p']
[2025-07-14T15:26:53.705+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask task_2
[2025-07-14T15:26:53.707+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=741) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-14T15:26:53.708+0000] {standard_task_runner.py:72} INFO - Started process 742 to run task
[2025-07-14T15:26:53.776+0000] {task_command.py:467} INFO - Running <TaskInstance: mon_premier_dag.task_2 manual__2025-07-14T15:26:47.949086+00:00 [running]> on host 97ccc390339b
[2025-07-14T15:26:53.935+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mon_premier_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-14T15:26:47.949086+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-14T15:26:47.949086+00:00'
[2025-07-14T15:26:53.936+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-14T15:26:53.956+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-14T15:26:53.959+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "This is my first DAG!"']
[2025-07-14T15:26:53.972+0000] {subprocess.py:99} INFO - Output:
[2025-07-14T15:26:53.974+0000] {subprocess.py:106} INFO - This is my first DAG!
[2025-07-14T15:26:53.975+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-14T15:26:54.037+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-14T15:26:54.038+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mon_premier_dag, task_id=task_2, run_id=manual__2025-07-14T15:26:47.949086+00:00, execution_date=20250714T152647, start_date=20250714T152653, end_date=20250714T152654
[2025-07-14T15:26:54.085+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-14T15:26:54.107+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-14T15:26:54.137+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
